.eslintrc.json

```diff
@@ -12,6 +12,8 @@
   ],
   "globals": {
     "console": true,
-    "module": true
+    "module": true,
```

Comment:
You can specify environment instead:
https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
thx, added